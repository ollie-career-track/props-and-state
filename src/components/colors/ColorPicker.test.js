import React from 'react';
import { shallow } from 'enzyme';
import ColorPicker from './ColorPicker';

describe('ColorPicker component', () => {
  it('renders color picker', () => {
    const wrapper = shallow(<ColorPicker colors={['green', 'blue', 'red', 'yellow']} selectColor={() => true} />);
    expect(wrapper).toMatchSnapshot();
  });
});
