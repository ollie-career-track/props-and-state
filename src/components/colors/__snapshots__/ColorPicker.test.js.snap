// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColorPicker component renders color picker 1`] = `
<Fragment>
  <button
    key="green"
    onClick={[Function]}
  >
    green
  </button>
  <button
    key="blue"
    onClick={[Function]}
  >
    blue
  </button>
  <button
    key="red"
    onClick={[Function]}
  >
    red
  </button>
  <button
    key="yellow"
    onClick={[Function]}
  >
    yellow
  </button>
</Fragment>
`;
